addons/account_edi_facturx/test_file/test_facturx.xml

```diff
@@ -42,6 +42,9 @@ Conference room table</ram:Name>
                 <ram:SpecifiedTradeSettlementLineMonetarySummation>
                     <ram:LineTotalAmount currencyID="USD">3210.00</ram:LineTotalAmount>
                 </ram:SpecifiedTradeSettlementLineMonetarySummation>
+                <ram:ApplicableTradeTax>
+                    <ram:RateApplicablePercent>0.0</ram:RateApplicablePercent>
+                </ram:ApplicableTradeTax>
```

Comment:
this is weird... There is no zero amount tax in generic_coa. Where does it come from?

Reply:
I am [creating two taxes](https://github.com/odoo/odoo/pull/75738/files#diff-0b76d184eb25457af66f96d17a64eae5ae07bdbe25015a42f62538307d40f7fcR253-R257) in my new test, and the amount is zero so that the total is not modified for other tests.
However, if this is too weird, do you have a better idea for writing this test?

Reply:
I'm ok with that then :)